pkg/oci/remote/remote.go

```diff
@@ -116,31 +116,31 @@ func suffixTag(ref name.Reference, suffix string, o *options) (name.Tag, error)
 	return o.TargetRepository.Tag(normalize(h, o.TagPrefix, suffix)), nil
 }
 
-type digestable interface {
+type digestible interface {
```

Comment:
API?

Reply:
I think this one was correct before - it's not a "real word" but we use the "-able" suffix for interfaces.